anjali/sb-pt-cont/6.js

```diff
@@ -0,0 +1,61 @@
+// https://levelbuilder-studio.code.org/s/sb-pt-cont/lessons/1/levels/6
+/*
+It's time to start creating your personal Story Board!
+Do This:
+Add some costumes to your project by clicking the costumes tab.
+Add code to the workspace to make a sprite that represents YOU!
+*/
+if (!validationProps.successCriteria) {
+  validationProps.successCriteria = {
+    starterSprite: false,
+    activeSpeech: false
+  };
+}
+
+// Helper variables
+var spriteIds = getSpriteIdsInUse();
+
+// Check for at least 1 sprite
+if (spriteIds.length >= 1) {
+  validationProps.successCriteria.starterSprite = true;
+}
+
+/*
+for (var i = 0; i < spriteIds.length; i++) {
+  if(getProp({ id: i }, "timeOut")){  
+    validationProps.activeSpeech = true;
+  }
+}
+*/
+
+// Set success time if success
+if (validationProps.successCriteria.starterSprite &&
+   !validationProps.successTime)
+{
+  validationProps.successTime = World.frameCount;
```

Comment:
this is just checking that there is at least one sprite? Should we also check something about the costume?

Reply:
This mini-lesson is about behaviors, so I don't think there's a need to check for different costumes.

Reply:
check that sprite costume is not starting (pizza)